webapp/components/needs_team.jsx

```diff
@@ -42,20 +42,17 @@ export default class NeedsTeam extends React.Component {
         this.onChanged = this.onChanged.bind(this);
 
         this.state = {
-            profiles: UserStore.getProfiles(),
```

Comment:
Yes, this component will not render again in that case but it doesn't need to since the only child that used the profiles was PostsView, which will still re-render when it gets an event about profiles changing. I just moved this code down into PostsView since there was no reason to have it this high up. None of the other children of NeedsTeam were using the profiles.

Having the profiles here was causing unneeded rendering of NeedsTeam which was really slow since it's such a high-level component.